templates/timesyncd.conf.j2

```diff
@@ -0,0 +1,6 @@
+# vi: ft=dosini:
```

Comment:
What is there to edit, is your role not complete? I don't think optimizations for your local editor belong into the template file.